hack/lib/golang.sh

```diff
@@ -68,6 +81,20 @@ else
   fi
   readonly KUBE_SERVER_PLATFORMS
 
+  # The node platforms we build for
+  KUBE_NODE_PLATFORMS=(
+    linux/amd64
+    linux/arm
+    linux/arm64
+    linux/s390x
+    windows/amd64
+    windows/386
```

Comment:
probably not, though apparently there are still `windows/386` users out there. x-ref https://github.com/kubernetes/kubernetes/issues/28632